mods/default/locale/default.fr.tr

```diff
@@ -67,12 +67,12 @@ Obsidian Block=Block d'obsidienne
 Dirt=Terre
 Dirt with Grass=Terre avec de l'herbe
 Dirt with Grass and Footsteps=Terre avec de l'herbe et des traces de pas
-Dirt with Savanna Grass=
+Dirt with Savanna Grass=Terre avec de l'herbe de savanne
 Dirt with Snow=Terre avec de la neige
 Dirt with Rainforest Litter=Terre avec sol de forêt humide
 Dirt with Coniferous Litter=Terre avec sol de forêt de conifère
-Savanna Dirt=
-Savanna Dirt with Savanna Grass=
+Savanna Dirt=Terre de la savane
```

Comment:
Typo: Should be `savanne`?

Reply:
Actually, it should be `savane`... I've changed it everywhere in the following commit.